### PR TITLE
Update project for repo rename

### PR DESCRIPTION
**Pull request by moolitayer**
Testing...

### CODE REVIEW

**Comment by simon3z** [comment]
@moolitayer when will we start using this repo for the gem and ManageIQ?

**Comment by moolitayer** [comment]
I was thinking to handle this next time we make a release for it

**Comment by cben** [comment]
@enoodle and I want to work on a new release of this gem.

@simon3z, was this rename just to reflect new repo name?  The gem is not getting renamed, right?

**Comment by simon3z** [comment]
> @simon3z, was this rename just to reflect new repo name? The gem is not getting renamed, right?

The name of the gem is defined in the `gemspec` file.

**Comment by moolitayer** [comment]
@cben please review

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# image-inspector-client
+# image-inspector-client-ruby
```
Comment (cben) [inline README.md]: I think the title need not reflect the repo name (github already shows that), the gem name is more interesting to the reader.  Perhaps:
# image-inspector-client Ruby Gem

Comment (cben) [reply README.md]: hmm, that was serious bikeshedding on my part :bike: :house:, doesn't matter.

> *What’s in a name? That which we call image-inspector-client
> By any other name would Look as Good To Me :+1:;*

Comment (moolitayer) [reply README.md]: I think keeping `image-inspector-client` is a good idea now that you mention it (due to repo name!=gem name). Does that work for you?

Comment (cben) [reply README.md]: Sure.

**Review by cben — APPROVED** [review]
LGTM

**Pull request merged**